src/renderer/components/dialogs/MailtoDialog/styles.module.scss

```diff
@@ -0,0 +1,10 @@
+.MailtoDialog {
+  height: 100%;
+}
+
+.MailtoDialogBody {
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 0px;
```

Comment:
I think these should be in snake case to be consistent with other styles we've got?